.travis.yml

```diff
@@ -14,7 +14,9 @@ install:
     - travis_retry docker-compose --file docker-compose.yaml --file docker-compose.test.yaml build
 
 script:
-    - .travis/test/smoke
+    - .travis/smoke-test.sh
+    - docker-compose --file docker-compose.yaml --file docker-compose.test.yaml run app composer validate --strict
+    - docker-compose --file docker-compose.yaml --file docker-compose.test.yaml run app composer check-platform-reqs
```

Comment:
the `test` image is derived directly from `prod` so it makes sense to check this inside it